website/content/en/preview/getting-started/_index.md

```diff
@@ -82,8 +82,9 @@ You can add the details of the newly created EKS cluster to your local kubectl a
 ```bash
 aws eks update-kubeconfig --region ${AWS_DEFAULT_REGION} --name ${CLUSTER_NAME}
 ```
+Verify that you have a node with ready status by running `kubectl get nodes`.
```

Comment:
eksctl verifies that the nodes are `Ready` as part of creating the nodegroup so you don't need this.

Reply:
That makes sense, removed it.